src/react-forms.js

```diff
@@ -418,6 +416,7 @@ class ReactForms extends Component {
     }
 
     await this.startSubmit();
+    this.blockSubmission = true;
```

Comment:
Could probably even put this before the `await this.startSubmit()`

There will be at least 20ms where the user could increase the submit count again but it wouldn't actually submit

Reply:
yeah true